cmd/jaeger/internal/all-in-one.yaml

```
@@ -5,7 +5,17 @@ service:
       receivers: [otlp, jaeger, zipkin]
       processors: [batch]
       exporters: [jaeger_storage_exporter]
-
+  telemetry:
```

```suggestion
  telemetry:
    resource:
      service.name: jaeger
```
This might work without the hack